From 9252de815e4443d1b617211f38685afe7c0f6b0e Mon Sep 17 00:00:00 2001
From: Matthias Klumpp <matthias@tenstral.net>
Date: Sat, 20 Oct 2018 20:20:09 +0200
Subject: [PATCH] Explicitly include string.h in as-tag.c

This apparently is necessary to fix an FTBFS issue with older GCC
releases. Resolves #207
---
 src/as-tag.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/as-tag.c b/src/as-tag.c
index 6afba621..91076e69 100644
--- a/src/as-tag.c
+++ b/src/as-tag.c
@@ -32,6 +32,8 @@
 
 #include "as-tag.h"
 
+#include <string.h>
+
 #ifdef __clang__
 #pragma clang diagnostic ignored "-Wmissing-field-initializers"
 #endif
